ambry-coordinator/src/test/java/com.github.ambry.coordinator/MockDataNode.java

```diff
@@ -35,36 +32,16 @@
 public class MockDataNode {
   private DataNodeId dataNodeId;
 
-  private Map<BlobId, MaterializedBlob> blobs;
```

Comment:
Yeah, I can get your point. But I feel that this is not a sweeping or a very big change. Very trivial fix in my opinion.